checks.d/kong.py

```diff
@@ -0,0 +1,71 @@
+# stdlib
+import urlparse
+
+# 3rd party
+import requests
+import simplejson as json
+
+# project
+from checks import AgentCheck
+from util import headers
+
+
+class Kong(AgentCheck):
+    """ collects metrics for Kong """
+    def check(self, instance):
+        metrics = self._fetch_data(instance)
+        for row in metrics:
+            try:
+                name, value, tags = row
+                self.gauge(name, value, tags)
+            except Exception:
+                self.log.error(u'Could not submit metric: %s', row)
+
+    def _fetch_data(self, instance):
+        if 'kong_status_url' not in instance:
+            raise Exception('missing "kong_status_url" value')
+        tags = instance.get('tags', [])
+        url = instance.get('kong_status_url')
+
+        parsed_url = urlparse.urlparse(url)
+        host = parsed_url.hostname
+        port = parsed_url.port or 8001
```

Comment:
I'm not sure I understand what you mean.
I think there are two cases : 
- `kong_status_url: http://something:port/status`, and `parsed_url.port` returns `port` as expected.
-  `kong_status_url: http://something/status`, and `parsed_url.port` returns `None`, and there is (I think) a small issue, because the current check tags with `kong_port: 8001` whereas `response = requests.get(url, headers=headers(self.agentConfig))` tries to connect on the port 80 (default HTTP port), contradicting the tag.

If you want to have a default for the `kong_port`, you could instead of asking for an URL in the config file (`kong.yaml.example`) ask for a `port`(set a default to 8001) and a `host` and then build the URL in the check `url = 'http://{}:{}/status'.format(host, port)`.
It would only be possible if the URL is always the same (the `/status` part).

It's really a nitpick (I guess every user of the check will specify the port), but please let me know what you think about it.